File: TransCoder_tokenized_test_set_functions/java/SORT_3_INTEGERS_WITHOUT_USING_CONDITION_USING_MAX_FUNCTION.java
static void printSorted ( int a, int b, int c ) {
  int get_max = Math . max ( a, Math . max ( b, c ) );
  int get_min = - Math . max ( - a, Math . max ( - b, - c ) );
  int get_mid = ( a + b + c ) - ( get_max + get_min );
  System . out . print ( get_min + " " + get_mid + " " + get_max );
}

File: TransCoder_tokenized_test_set_functions/python/SORT_3_INTEGERS_WITHOUT_USING_CONDITION_USING_MAX_FUNCTION.py
def printSorted ( a, b, c ) :
    get_max = max ( a, max ( b, c ) )
    get_min = - max ( - a, max ( - b, - c ) )
    get_mid = ( a + b + c ) - ( get_max + get_min )
    print ( get_min, " ", get_mid, " ", get_max )
